### ISSUE TITLE
Viewport of Login Controller

### ISSUE THREAD

**Issue by dancuconati** [forcedotcom/SalesforceMobileSDK-CordovaPlugin#404]
I'm having a problem with size of the Salesforce login controller in landscape orientation on an iPad. You can see that the right side of the picture attached has a blue background and the view controller doesn't extend the whole screen.
![landscape](https://user-images.githubusercontent.com/20344292/40146565-fab8e0a4-5933-11e8-83bd-17222ab374ad.PNG)

Here is what we have overridden in our AppDelegate+SalesforceHybridSDK.m:
`#define IDIOM    UI_USER_INTERFACE_IDIOM()
#define IPAD     UIUserInterfaceIdiomPad#if __IPHONE_OS_VERSION_MAX_ALLOWED < 90000
- (NSUInteger)application:(UIApplication*)application supportedInterfaceOrientationsForWindow:(UIWindow*)window
#else //CB-12098.  Defaults to UIInterfaceOrientationMask for iOS 9+
- (UIInterfaceOrientationMask)application:(UIApplication*)application supportedInterfaceOrientationsForWindow:(UIWindow*)window
#endif
{
    // iPhone doesn't support upside down by default, while the iPad does.  Override to allow all orientations always, and let the root view controller decide what's allowed (the supported orientations mask gets intersected).
    if ( IDIOM == IPAD ) {
        return UIInterfaceOrientationMaskAll;
    } else {
        return UIInterfaceOrientationMaskPortrait;
    }
}`

We are also setting the following on the plist file: 
![plist](https://user-images.githubusercontent.com/20344292/40146700-939af3ac-5934-11e8-873f-023403eb1f82.png)

Is there a way to make the login view controller extend the whole screen in landscape or is this a bug?

Thanks,

**Comment by bhariharan** [comment]
I don't think we support the login view controller in landscape mode, it's meant to be locked down in portrait mode. @trooper2013

**Comment by trooper2013** [comment]
@dancuconati As @bhariharan mentioned, we do not officially support landscape mode. I understand that this behavior may not be controllable on iPad devices.  Do lookout for subsequent PR's in the next few days that resolves the issue. Alternatively you could extend the SFLoginViewController and set the autoresize mask, allow auto resize of subviews  for the loginViewController and maybe the WKWebview as well.  [Review this discussion](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/issues/2477)
